Swift/BRCrypto/BRCryptoTransfer.swift

```diff
@@ -321,7 +321,9 @@ public enum TransferState {
             confirmation: TransferConfirmation (blockNumber: core.u.included.blockNumber,
                                                 transactionIndex: core.u.included.transactionIndex,
                                                 timestamp: core.u.included.timestamp,
-                                                fee: core.u.included.fee.map { Amount (core: $0, take: false) }))
+                                                fee: core.u.included.feeBasis
```

Comment:
Leaking `feeBasis` here?

Reply:
Is it?  The caller to `init` owns `core` and thus `feeBasis`.  The `core.u.included.feeBasis` doesn't take the reference.  Am I missing it?

Reply:
The C code has taken a reference to the fee basis before passing it up to the Swift/Java layer. You then get the fee out of it but you don't give the feebasis reference up (that I notice, maybe I am missing it?).

Reply:
Addressed w/ cryptoTransferStateRelease() in the TransferState initializer.  Does that appear sufficient?